### PR TITLE
DEVOP-124: ci: added dependabot configuration

### PR DESCRIPTION
**Pull request by DaisyDomergue**

### CODE REVIEW

**Comment by DaisyDomergue** [comment]
I only added gitaction configurartion I didn't include the docker specification in the dependabot because I don't think this repo utilizes docker

**Review by kare — APPROVED** [review]

**Comment by kare** [comment]
@DaisyDomergue Note the title! Also try to remember to use conventional commits with the actual commit messages.

File: .github/dependabot.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      interval: daily
+      time: "08:00"
+      timezone: Europe/Helsinki
+    commit-message:
+      prefix: ci
+      include: scope
+    labels:
+      - ci
+    reviewers:
+      - kare
```
Comment (kare) [inline .github/dependabot.yaml:21]: Change to `harbu`

File: .github/dependabot.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    assignees:
+      - DaisyDomergue
```
Comment (kare) [inline .github/dependabot.yaml:23]: Change to `timoxley`

**Comment by DaisyDomergue** [comment]
changed the assignee and the reviewer

**Comment by kare** [comment]
@DaisyDomergue Sorry my bad, I thought that I was looking at the Node/Javascript Dependabot config. So GitHub Actions config as usual. Reviewers/assignees I mentioned are for Node/Javascript config. So at the moment it's missing the Node section.

**Review by kare — CHANGES_REQUESTED** [review]
Also note that you need a Dependabot Node/Javascript section for each type/directory:
- ts
- react
- node.js

**Comment by DaisyDomergue** [comment]
> Also note that you need a Dependabot Node/Javascript section for each type/directory:
> 
>     * ts
> 
>     * react
> 
>     * node.js

I don't we require such configurations since all of them use npm as the main package manager I think that is enough.

File: .github/dependabot.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  - package-ecosystem: "npm"
+    directory: "/"
```
Comment (kare) [inline .github/dependabot.yaml:25]: Path is missing

File: .github/dependabot.yaml [hunk 4]
```diff
@@ -35,4 +35,34 @@ updates:
...
\ No newline at end of file
+      - timoxley
+  - package-ecosystem: "npm"
+    directory: "/react/"
+    schedule:
+      interval: daily
+      time: "08:00"
+      timezone: Europe/Helsinki
+    commit-message:
+      prefix: Build
```
Comment (kare) [inline .github/dependabot.yaml]: Label is `build`

**Review by kare — CHANGES_REQUESTED** [review]
Needs some attention on details.

File: .github/dependabot.yaml [hunk 5]
```diff
@@ -35,4 +35,34 @@ updates:
     reviewers:
       - harbu
     assignees:
-      - timoxley
\ No newline at end of file
+      - timoxley
```
[the remark this answers is not in the document]
Comment (DaisyDomergue) [reply .github/dependabot.yaml:38]: I have already fixed it in the second commit [https://github.com/streamr-dev/eslint-config-streamr/pull/23/commits/60149f89632e7c6b3520beecae7c2f9ae48d624e](second commit) can you check that one please

**Review by kare — APPROVED** [review]
LGTM

**Pull request merged**